test/print_graph_tests.cc

```diff
@@ -158,11 +169,11 @@ TEST_CASE_METHOD(test_utils::runtime_fixture, "full graph is printed if CELERITY
 
 	SECTION("task graph") {
 		const auto* expected =
-		    "digraph G {label=\"Task Graph\" 0[shape=ellipse label=<T0<br/><b>epoch</b>>];1[shape=box style=rounded label=<T1 \"full_graph_printing_17\" "
```

Comment:
Something I did in some other printing tests IIRC is remove the `UKN` (meaning one has to be careful with the names), or simply remove the task name entirely (since I believe we no longer support any SYCL implementation which requires names).

Reply:
Done - I have removed most kernel names altogether (which requires a patch to `handler.h` to work correctly).